test/unit/answer_test.rb

```diff
@@ -46,6 +46,6 @@ class AnswerTest < ActiveSupport::TestCase
 
   test "should list comments in descending order" do
     answer = answers(:one)
-    assert_equal answer.comments.first, DrupalComment.last
+    assert_equal answer.comments.first, Comment.last
```

Comment:
Here, as I described in my longer comment, use `Comment.last` since we can no longer expect default ordering.